test-adapter/src/main/scala/org/scalajs/testing/adapter/HTMLRunnerBuilder.scala

```diff
@@ -114,4 +99,20 @@ object HTMLRunnerBuilder {
     case '&' => "&amp;"
     case c   => c.toString()
   }
+
+  private def loadCSS(): String = {
+    val builder = new StringBuilder
+    val buf = CharBuffer.allocate(1024)
+    val in = new InputStreamReader(getClass.getResourceAsStream("test-runner.css"), UTF_8)
+    try {
+      while (in.read(buf) != -1) {
+        buf.flip()
+        builder.append(buf)
+        buf.clear()
+      }
+      builder.toString
+    } finally {
+      in.close()
+    }
+  }
```

Comment:
I really do not like this verbose, low-level thing here. The only alternative I found was to write to a jimfs file first (and read back). That would make use depend on jimfs.

Reply:
In Java 9 that would be
```scala
val in = getClass.getResourceAsStream("test-runner.css")
try new String(in.readAllBytes(), UTF_8)
finally in.close()
```
but sticking to JDK 8 APIs, I believe the existing version is best.